src/documents/articles/interview-with-travis-leithead.html.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Web Components and Microsoft Edge
+authors: [travis_leithead, erik_isaksen]
+date: 2015-07-08
+original_date: 2015-07-08
+image: interview-with-travis-leithead.jpg
+category: articles
+layout: single
+tags: ['Interview', 'Edge Browser']
```

Comment:
Since we don't have any post tagged with `Edge Browser`, you can replace that with `Browser Support`.

http://webcomponents.org/tags/browser-support/